### ISSUE TITLE
Submit Glimpse to the Microsoft Store

### ISSUE THREAD

**Issue by TrechNex** [glimpse-editor/Glimpse#180]
A not small number of schools and businesses use "Windows 10 S", which restricts their ability to install or run software from third party sources.

Providing Glimpse for free through the app store on Windows would neatly get around this problem within organizations that permit the installation of applications using that method.

The other benefit from our project's perspective is we would be advertised through a deployment channel that is automatically installed on every Windows 10 machine.

**Comment by TrechNex** [comment]
The idea has been mooted before that we could take Paint.NET's approach of charging for downloads through the Microsoft Store and distributing for free via their own website.

I disagree on the grounds that this would block people who are using centrally-managed machines from installing our software because they would be unable to provide their own payment details (and in an enteprise setting may even be required to raise a business case!).

**Comment by TrechNex** [comment]
Tangentially related to #181

**Comment by TrechNex** [comment]
The WiX Toolset installer is now complete. I'm waiting on some final changes before rebuilding for 0.1.0 release.

I have also started our Windows Store submission. Also looked up changes that might be needed for our application to be eligible: https://docs.microsoft.com/en-us/windows/msix/packaging-tool/create-app-package-msi-vm

**Comment by TrechNex** [comment]
Postponed to next release because of problems fixing the MSIX file we need to complete the submission.

**Comment by TrechNex** [comment]
Brought back in as 0.1.x candidate as I think the MSIX problem we currently have (the EXE not starting in the right working directory) can be solved if I dedicate some time to fixing it.

**Comment by TrechNex** [comment]
I'm closing this and clearing milestones, because I think distributing our application through Windows Store would not be appropriate until we reach the 1.0 stage.

There is also no guarantee that the UI rewrite would be compatible with Windows Store, so we need to wait on more information before we commit to doing this.

**Comment by TrechNex** [comment]
Re-opening as I think it would be cool to make this happen. In theory the only blocker is dedicating time to figuring out how PSFlaunch works.

Setting as 0.2.0 candidate, but may get pushed out to 0.3.0 as the macOS port has priority.

**Comment by TrechNex** [comment]
Following this issue where someone upstream is proving out MSIX installer support.
https://gitlab.gnome.org/GNOME/gimp/issues/4000

**Comment by TrechNex** [comment]
Bringing forward as people are having real problems running both Glimpse Image Editor and GNU Image Manipulation Program on locked down work machines with overly-jumpy anti-virus.

We're going to be working on an MSIX anyway, so also providing a mechanism where people can download a security-audited and Microsoft-signed version of the application would be another way of ensuring we fix that problem.

**Comment by TrechNex** [comment]
Kicking to 0.2.0 because I think we should wait until we rebase on 2.10.20

**Comment by TrechNex** [comment]
After discussing, we're closing this issue because it would require one specific individual to maintain and run this.
